packages/owebl/owebl.0.1/opam

```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+name: "owebl"
+version: "0.1"
+maintainer: "Phil Eaton <me@eatonphil.com>"
+author: "Phil Eaton <me@eatonphil.com>"
+homepage: "http://meetowebl.com"
+bug-reports: "https://github.com/eatonphil/owebl/issues"
+license: "BSD"
+dev-repo: "https://github.com/eatonphil/owebl.git"
+build: [
+    [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "owebl"]
+depends: ["ocamlfind" "str" "unix"]
```

Comment:
The `depends` field is for OPAM packages, and not for `ocamlfind` ones (which are one level down -- an OPAM package can contain multiple `ocamlfind` packages).  Including them leads to:

```
Package is unavailable on this configuration, skipping:
The following dependencies couldn't be met:
  - owebl -> unix
Your request can't be satisfied:
  - No package named unix found.
```

You can fix this just by removing `str` and `unix` from the `depends:` field above.